src/components/Products.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+
+const Products = () => {
+    const [products, setProducts] = useState([]);
+
+    const fetchProducts = async () => {
+        try {
+            const response = await axios.get('/products');
+            console.log(response)
+            const result = await response.json();
+            console.log(result)
+            setProducts(result);
+        } catch (error) {
+            console.log("Trouble gathering products!", error)
+        }
+    }
+
+    // useEffect(fetchProducts, []); 
```

Comment:
sounds like this is in a PR, but let's use this line instead of calling it repeatedly.

Reply:
complete